### PR TITLE
New Project 

### PR DESCRIPTION
**Pull request by Tayo-Wunmi**

### CODE REVIEW

File: WEEK ONE/Challenge 1, Even Or Odd.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+if fav_num % 2 == 0:
+    print('Your number favourite is an even number')
+
+else:
+    print('Your number favourite is an odd number')
+
+print("Thank you for playing the number identificator. Hope you had fun! "
+      "\n"
+      "Please give comment or feedback: ")
```
Comment (Del-sama) [inline WEEK ONE/Challenge 1, Even Or Odd.py]: What happens when I type something like `Nice` as my comment?

Comment (Del-sama) [reply WEEK ONE/Challenge 1, Even Or Odd.py]: The last line looks like it expects an input from the user, but there is no `input` function

Comment (Tayo-Wunmi) [reply WEEK ONE/Challenge 1, Even Or Odd.py]: Noted. 

I will add the input function for the feedback, and also an acknowledgement of feedback

Comment (Tayo-Wunmi) [reply WEEK ONE/Challenge 1, Even Or Odd.py]: Code Updated!

File: WEEK ONE/Challenge 1, Even Or Odd.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+if fav_num % 2 == 0:
+    print('Your number favourite is an even number')
+
+else:
+    print('Your number favourite is an odd number')
+
+print("Thank you for playing the number identificator. Hope you had fun!")
+
+comment = (input("Please give comment or feedback: "))
+print("Feedback received")
```
Comment (Del-sama) [inline WEEK ONE/Challenge 1, Even Or Odd.py:14]: It's good practice to leave an empty line after the last line of code.

**Review by Del-sama — APPROVED** [review]

**Pull request merged**